datalad/distributed/ora_remote.py

```diff
@@ -205,7 +205,13 @@ def rename(self, src, dst):
         src.rename(dst)
 
     def remove(self, path):
-        path.unlink()
+        try:
+            path.unlink()
+        except PermissionError as e:
+            raise RIARemoteError(str(e) + os.linesep +
+                                 "No support for fixing your permissions. Note,"
+                                 " that write permissions to the directory are "
+                                 "required to drop remote content.")
```

Comment:
```suggestion
                                 "Note: Write permissions for a key's parent directory are "
                                 "also required to drop content.")
```

Reply:
I suggested a change. It is not at all given that the permissions are "mine", or that they could be fixed with my level of authorization.